src/git/config.go

```diff
@@ -141,9 +141,7 @@ func GetRemoteUpstreamURL() string {
 // GetURLHostname returns the hostname contained within the given Git URL.
 func GetURLHostname(url string) string {
 	hostnameRegex, err := regexp.Compile("(^[^:]*://([^@]*@)?|git@)([^/:]+).*")
-	if err != nil {
-		log.Fatal("Error compiling hostname regular expression: ", err)
-	}
+	logs.FatalOn(err, "Error compiling hostname regular expression: ", err)
```

Comment:
By wrapping errors we don't need to supply the err again. This is because the wrapped error message will include the original error message
```golang
logs.FatalOn(err, "Error compiling hostname regular expression: ", err)
// becomes
logs.FatalOnWrap(err, "Error compiling hostname regular expression")

logs.FatalfOn(err, "Error compiling configuration regular expression (%s): %v", toMatch, err)
// becomes
logs. FatalOnWrapf(err, "Error compiling configuration regular expression: %s", toMatch)
```